ci-builder/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # This Dockerfile is used in openshift CI
 FROM quay.io/fedora/fedora:37
```

Comment:
Upgrading fedora to a supported version might make sense too.

Reply:
done